spec/routing/users_routing_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe UsersController, type: :routing do
+  describe "routing" do
+
+    it "routes to #index" do
+      expect(:get => "/users").to route_to("users#index")
+    end
+
+    it "routes to #new" do
+      expect(:get => "/users/new").to route_to("users#new")
+    end
+
+    it "routes to #show" do
```

Comment:
I see there is a test for this but I can't seem to run these routes in the production app.
Is this deliberate?

Reply:
it might have been changed, please recheck